core/templates/dev/head/pages/topic_viewer/topic_viewer.html

```diff
@@ -0,0 +1,155 @@
+        </a>
+      </li>
+    </ul>
+    <div ng-if="activeTab === 'story'">
+      <div ng-if="canonicalStoryList.length === 0" class="oppia-dashboard-empty-text">
+        <em>There are no stories to display.</em>
+      </div>
+
+      <div ng-if="activeTab === 'story'">
+        <md-card ng-repeat="canonicalStory in canonicalStoryList" class="oppia-story-summary-tile">
+          <canonical-story-summary-tile story-id="canonicalStory.id"
+                                        title="canonicalStory.title"
+                                        description="canonicalStory.description">
+          </canonical-story-summary-tile>
+        </md-card>
+      </div>
+    </div>
+    <div ng-if="!userIsLoggedIn" class="oppia-login-banner">
+      <div><img ng-src="<[oppiaBlackImgUrl]>" alt="" style="width: 70px; margin: 10px 0"></div>
+      <div>Sign in to save your progress!</div>
+      <div><button class="btn oppia-navbar-button"><span translate="I18N_TOPNAV_SIGN_IN"></span></button></div>
+    </div>
+  </div>
+  <style>
+    .oppia-login-banner {
+      border: #009688 solid 3px;
+      border-radius: 5px;
+      max-width: 60%;
+      position: fixed;
+      bottom: -5px;
+      left: 35%;
+      margin: 0 auto;
+    }
+    .oppia-login-banner div {
+      display: inline;
+      font-size: 1.4em;
+      margin: 20px;
+    }
+
+    .oppia-topic-viewer-container .oppia-topic-name {
+      color: #01645c;
+      font-family: 'Capriola', 'Roboto', Arial, sans-serif;
+      font-size: 3em;
+      margin-top: 0px;
+      margin-bottom: 25px;
+      padding-top: 69.5px;
+      text-align: center;
+    }
+
+    .oppia-story-summary-tile {
+      background-color: #fff;
+      cursor: pointer;
+      display: inline-block;
+      margin: 25px;
+      padding: 0;
+      position: relative;
+      text-align: left;
+      transition: all 300ms;
+      vertical-align: top;
+      -webkit-transition: all 300ms;
+      white-space: normal;
+      width: 268px;
+    }
+
+    .oppia-topic-viewer-container {
+      max-width: 80%;
+      color: black;
+      text-decoration: none;
+      margin: 0 auto;
+      font-family: 'Roboto', Arial, sans-serif;
+    }
+
+    .oppia-dashboard-tabs li {
+      display: -webkit-flex;
+      display: inline-flex;
+      margin-bottom: 0;
+      width: 200px;
+      background-color: #7e7e7e;
+    }
+
+    .oppia-dashboard-tabs .oppia-dashboard-tabs-text {
+      color: white;
+      font-size: 1em;
+      padding: 10px;
+      text-decoration: none;
+      text-transform: uppercase;
+      width: 100%;
+    }
+
+    .oppia-dashboard-tabs-active .oppia-dashboard-tabs-text,
+    .oppia-dashboard-tabs-active .oppia-dashboard-tabs-text:hover {
+      border-top: 4px solid #009688;
+      color: #009688;
+      background-color: #e7e7e2;
+    }
+
+    .oppia-dashboard-tabs .oppia-dashboard-tabs-text:hover {
```

Comment:
For most of these styles, why .oppia-dashboard?

Reply:
Chnged it to .oppia-topic-viewer